x-pack/plugins/alerting/server/task_runner/execution_handler.ts

```diff
@@ -184,59 +185,98 @@ export class ExecutionHandler<
         ruleRunMetricsStore.incrementNumberOfTriggeredActions();
         ruleRunMetricsStore.incrementNumberOfTriggeredActionsByConnectorType(actionTypeId);
 
-        const actionToRun = {
-          ...action,
-          params: injectActionParams({
-            ruleId,
-            spaceId,
-            actionTypeId,
-            actionParams: transformActionParams({
-              actionsPlugin,
-              alertId: ruleId,
-              alertType: this.ruleType.id,
-              actionTypeId,
-              alertName: this.rule.name,
+        if (alert) {
```

Comment:
**nit:** to make this code more descriptive, can we make the `executables` contain an attribute determining if it's a summary or not?
```suggestion
        if (!isSummary) {
```

Reply:
we already have it `this.isSummaryAction(action)` but have to add alert object in the if case anyway to avoid ts errors

Reply:
Done in [#3d137a1](https://github.com/elastic/kibana/pull/147360/commits/3d137a13893d66c7b29ac93e20446f4fefbba12c)